### DISCUSSION TITLE
TuringCourses/short_term/rust/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#109] [General]
# TuringCourses/short_term/rust/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/short_term/rust/

**Comment by MTDickens** [comment]
建议先在 course.rs 上面，把基础部分全部刷一遍（最后的入门实践这部分可以跳过）。毕竟 Rust 的条条框框比 C/C++ 多得多，最好还是先用熟一点。

**Reply by MTDickens** [reply]
另外今年没了，张寅老师的课程换成了腾讯的团队，讲的是强化学习
